presto-main/src/main/java/com/facebook/presto/operator/scalar/DateTimeFunctions.java

```diff
@@ -76,6 +76,7 @@ protected DateTimeFormatter load(Slice format)
     private static final int MILLISECONDS_IN_MINUTE = 60 * MILLISECONDS_IN_SECOND;
     private static final int MILLISECONDS_IN_HOUR = 60 * MILLISECONDS_IN_MINUTE;
     private static final int MILLISECONDS_IN_DAY = 24 * MILLISECONDS_IN_HOUR;
+    private static final int PIVOT_YEAR = 2020; // yy = 70 will correspond to 1970 but 69 to 2069
```

Comment:
On the flip side, if it was adaptive (based on the current year) some queries would change in behavior year-by-year. I believe it's better to expect any changes in query engine behavior by version, without adding a new variable (time of executing query).